zerver/webhooks/github/tests.py

```diff
@@ -611,13 +613,80 @@ def test_team_edited_with_unsupported_keys(self) -> None:
+        self.check_webhook("discussion__edited_title", expected_topic, expected_message)
+
+    def test_discussion_edited_body(self) -> None:
+        expected_message = "Niloth-p edited [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3):\n\n~~~ quote\nWriting good commit messages is an art, but it's also an important part of maintaining a clear and understandable project history. What are some tips and tricks you've learned for writing clear and concise commit messages? Do you have any favorite templates or formats?\r\nAny advice would be greatly appreciated!\n~~~"
+        self.check_webhook("discussion__edited_body", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_labeled(self) -> None:
+        expected_message = "Niloth-p added the **enhancement** label to [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3)."
+        self.check_webhook("discussion__labeled", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_locked(self) -> None:
+        expected_message = "Niloth-p **locked** [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3) as resolved."
+        self.check_webhook("discussion__locked", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_pinned(self) -> None:
+        expected_message = "Niloth-p **pinned** [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3)."
+        self.check_webhook("discussion__pinned", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_reopened(self) -> None:
+        expected_message = "Niloth-p **reopened** [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3)."
+        self.check_webhook("discussion__reopened", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_transferred(self) -> None:
+        expected_message = "Niloth-p transferred discussion #3 from **webhook-tester** to **another-webhook-tester** as [discussion #1](https://github.com/Niloth-p/webhook-tester/discussions/3)."
+        self.check_webhook("discussion__transferred", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_unlabeled(self) -> None:
+        expected_message = "Niloth-p removed the **enhancement** label from [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3)."
+        self.check_webhook("discussion__unlabeled", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_unlocked(self) -> None:
+        expected_message = "Niloth-p **unlocked** [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3)."
+        self.check_webhook("discussion__unlocked", TOPIC_DISCUSSION, expected_message)
+
+    def test_discussion_unpinned(self) -> None:
+        expected_message = "Niloth-p **unpinned** [discussion #3](https://github.com/Niloth-p/webhook-tester/discussions/3)."
```

Comment:
Is the use of bold for these something that we have as an existing standard in this integration?

I don't see any existing GitHub webhook events that do that -- they may bold the name, but never the action. 

I think it'd be better to be consistent with the existing styling? I think the bold would be quite distracting in practice.

Reply:
Great.